docs/examples/15_minutes_to_QCoDeS.ipynb

```diff
@@ -788,7 +804,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "After initialising the database we create the `Experiment` object. This object contains the name of the experiment and the sample, and the path of the database. You can  use `load_or_create_experiment` to find and return an experiment with the given experiment and sample name if it already exists, or create one if not found."
+    "After initialising the database we create the `Experiment` object. Note that if there is no database created or loaded before creating or loading an `Experiment`, by running `Experiment` modules that are related to creating an experiment, a database will be created or loaded (if already exists) according to the QCoDeS config. `Experiment` object contains the name of the experiment and the sample, and the path of the database. You can  use `load_or_create_experiment` to find and return an experiment with the given experiment and sample name if it already exists, or create one if not found."
```

Comment:
the "by running `Experiment` modules" is confusing - `Experiment` is class, what modules are meant?

i think this should be explained simpler and better - i suggest step-by-step explanation. first, just instroduce the experiment as it was before, and then in a paragraph-per-idea way explain what happens if there's an empty database without an experiment, or if there's no database but someone creates an experiment object.

Reply:
Sure. I added a little more clarifications. Please take a look and let me know if sounds reasonable.

I had a hard time typing initialize as initialise, though :)